packages/contracts-bedrock/src/L2/L2StandardBridge.sol

```diff
@@ -52,15 +52,19 @@ contract L2StandardBridge is StandardBridge, ISemver {
         bytes extraData
     );
 
-    /// @custom:semver 1.6.0
-    string public constant version = "1.6.0";
+    /// @custom:semver 1.7.0
+    string public constant version = "1.7.0";
 
     /// @notice Constructs the L2StandardBridge contract.
     /// @param _otherBridge Address of the L1StandardBridge.
     constructor(address payable _otherBridge)
         StandardBridge(payable(Predeploys.L2_CROSS_DOMAIN_MESSENGER), _otherBridge)
     { }
```

Comment:
We need to this for consistency with the L2ERC721Bridge. Though as is the suggestion might be modified by `forge fmt`:

```suggestion
    { initialize(); }
```

Reply:
ah good catch